src/components/DocExplorer.js

```diff
@@ -246,7 +255,12 @@ class SearchDoc extends React.Component {
     const matchedTypes = [];
     const matchedFields = [];
 
-    Object.keys(typeMap).forEach(typeName => {
+    const typeNames = Object.keys(typeMap);
+    for (const typeName of typeNames) {
+      if (matchedTypes.length + matchedFields.length === 100) {
```

Comment:
I think you want >=. It's possible that in one iteration you add both types and fields so that one iteration is 99 or less and the next is 101 or more